test/over_react_test/helper_components/sample_component.dart

```diff
@@ -62,12 +70,14 @@ class SampleComponent extends UiComponent2<SampleProps> {
   @override
   componentDidMount() {
     window.console.warn('Just a lil warning');
+    if (props.shouldErrorInMount) throw Error();
+    if (props.componentDidMountCallback != null) props.componentDidMountCallback();
```

Comment:
#fyi (what you have is fine), you can also do:
```dart
props.componentDidMountCallback?.call();
```

Reply:
Ohh that's so nice and clean, I like it!